[TW-1402] Doc data file support for performance testing (#5117)
postmanlabs/postman-docs: src/pages/docs/collections/performance-testing/testing-api-performance.md

The performance test runs on your computer and not in the cloud. When you start the test, the Collection Runner displays performance metrics in real time. You can [view performance test metrics](/docs/collections/performance-testing/performance-test-metrics/) such as the average response time, error rate, and throughput for all requests or individual requests. You can also [view details about errors](/docs/collections/performance-testing/performance-test-errors/) that occurred during the test to help identify the source of any performance issues.

## Contents

> * [Configuring a performance test](#configuring-a-performance-test)

Reviewer: ```suggestion
* [Configure a performance test](#configure-a-performance-test)
```

* [Virtual users and system resources](#virtual-users-and-system-resources)

> * [Viewing past performance runs](#viewing-past-performance-runs)

Reviewer: ```suggestion
* [View past performance runs](#view-past-performance-runs)
```

> ## Configuring a performance test

Reviewer: ```suggestion
## Configure a performance test
```

<img alt="Configuring a performance test" src="https://assets.postman.com/postman-docs/v10/performance-test-configure-v10-19.jpg"/>

> Your [Postman plan](https://www.postman.com/pricing/) gives you a limited number of performance runs you can use each month. A message will display in the Collection Runner when you're approaching your usage limit. Learn more about [resource usage](/docs/billing/resource-usage/#performance-test-usage) in Postman.

## Virtual users and system resources

The number of virtual users a performance test can simulate depends on available system resources and the collection used for the test. Using pre-request or test scripts will reduce the number of virtual users that can be simulated. The maximum number of virtual users is determined by your [Postman plan](https://www.postman.com/pricing/).

Use the following guidelines to understand how many virtual users you can simulate:

* A host with 8 CPU cores and 16 GB RAM can simulate up to 250 virtual users
* A host with 16 CPU cores and 32 GB RAM can simulate up to 500 virtual users

> Attempting to simulate a higher number of virtual users may cause inaccurate metrics and reduced throughput (requests per second). Postman recommends running a small test with 10 to 20 virtual users and observing system resource usage. You can use this information to estimate that maximum number of virtual users that can be simulated for your collection and host computer.

Reviewer: ```suggestion
Attempting to simulate a higher number of virtual users may cause inaccurate metrics and reduced throughput (requests per second). Try running a small test with 10 to 20 virtual users and observing system resource usage. You can use this information to estimate that maximum number of virtual users that can be simulated for your collection and host computer.
```

> If you have a problem with the performance testing feature, contact the [Postman support team](https://support.postman.com/hc/en-us). Make sure to attach your [Postman logs](https://support.postman.com/hc/en-us/articles/360025298633-How-to-get-logs-from-the-Postman-Desktop-app) in your support request.

> ## Viewing past performance runs

Reviewer: ```suggestion
## View past performance runs
```
